README.md

```diff
@@ -14,6 +14,13 @@ The Zotero translation server lets you use [Zotero translators](https://www.zote
 
 `npm start`
 
+### With docker
+
+```
+docker build -t zotero/translation-server-v2:latest .
+docker run -it -d -p 1969:1969 zotero/translation-server-v2:latest
```

Comment:
Why not choose a simplier name for the (local) tag?  I.e.
```
docker build -t translation-server-v2.
docker run -it -d -p 1969:1969 translation-server-v2
```

Reply:
Don't even need the v2. This repo will be renamed in the near future.